06_puppy/nickolee/mapStore.go

```diff
@@ -0,0 +1,46 @@
+package puppystorer
+
+import "fmt"
+
+// IncrementCounter increases the ID counter everytime a new Puppy is created to prevent overwrite issues
+// in DeletePuppy()
+func (ms *MapStore) IncrementCounter() {
```

Comment:
Make private or don't bother with it at all (it's a one liner).
Incrementing Counter is not something Store or MapStore should expose (it's more of an implementation detail).
In a sql for instance you'd use SERIAL or auto-increment and not have a code equivalent.

Reply:
Thanks for the tip. Done